op-plasma/commitment.go

```diff
@@ -27,6 +27,11 @@ func (c Keccak256Commitment) Encode() []byte {
 	return append([]byte{byte(Keccak256CommitmentType)}, c...)
 }
 
+// TxData adds an extra version byte to signal it's a commitment.
+func (c Keccak256Commitment) TxData(version uint8) []byte {
```

Comment:
Wouldn't keccak always be version 1?

Reply:
At the moment yes

Reply:
What is an example of where it wouldn't be version 1? I ask because can't we just not pass in a version byte as an arg?